src/main/java/de/cdiag/launchpadbackend/resources/UserController.java

```diff
@@ -35,4 +35,15 @@ public UserController(UserService userService) {
         }
 
     }
+
+    @GetMapping("error")
+    public void error() {
+        try {
+            final FileInputStream fi = new FileInputStream("doesNotExists.file");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *fi*. The resource is referenced in statements at the following line: 43. The resource closure statement is at line: 44. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by FileInputStream.readAllBytes. Either a) close *fi* in a try-finally block or b) close the resource by declaring *fi* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).